2-Draft-Accepted/RFC00NN-PowerShellCore-Installer-Module.md

```diff
@@ -0,0 +1,78 @@
+---
+RFC: RFC00NN
+Author: Joey Aiello
+Status: Draft-Accepted
+SupercededBy: N/A
+Version: 0.1
+Area: External Module
+Comments Due: 7/15/2017
+---
+
+# PowerShellCore Installer/Utility Module
+
+A module called `PowerShellCore` that ships on the Gallery and eases installation and configuration of PowerShell Core on Windows.
+
+## Motivation
+
+Installation and configuration of PowerShell Core on Windows is tricky due to the wide range of Windows versions that we support (7-10 and Server 2008R2-2016) and because of the usage of some "system" concepts shared by Windows PowerShell like the WinRM remoting plugin.
+
+## Goals
+
+* I can install PowerShell Core across all supported versions of Windows and Windows PowerShell *at scale*.
+* I can install PowerShell Core on certain versions of Windows and Windows PowerShell without remembering highly arcane commands or manually downloading a package from a well-known location.
+* I can uninstall PowerShell Core using Windows PowerShell
+* I can register PowerShell Core as a WinRM remoting endpoint across all supported versions of Windows and Windows PowerShell *at scale*.
+
+## Specification
+
+Ship a module on Gallery called `PowerShellCore` which contains the following cmdlets:
+
+* Install-PowerShellCore
+  * Download and install the latest PowerShell Core MSI and install it using the MSI PackageManagement provider
```

Comment:
Download and install a *specific version* of PowerShell Core, not just the latest?

```powershell
Install-PowerShellCore -Version 6.0.0-beta2
```